stories/constants/colors.stories.tsx

```diff
@@ -1,54 +1,26 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import { SPACER } from 'src/constants';
-import { Typography } from 'src/shared-components';
-import * as ALL_COLORS from 'src/constants/colors';
+import { useTheme } from 'emotion-theming';
 
 import Color from './color';
 
 const MainContainer = styled.div`
-  padding: ${SPACER.xlarge};
-`;
-
-const ColorsContainer = styled.div`
-  margin: ${SPACER.xlarge} 0;
   display: flex;
   flex-flow: row wrap;
-  justify-content: space-around;
+  justify-content: flex-start;
   align-items: center;
 `;
 
-/* eslint-disable @typescript-eslint/no-unused-vars */
-const {
-  default: defaultExport,
-  COLORS_PROP_TYPES,
-  ...VALID_COLORS
-} = ALL_COLORS;
-/* eslint-enable @typescript-eslint/no-unused-vars */
-
 export const COLORS = () => {
-  const colorKeys = Object.keys(VALID_COLORS) as Array<
-    keyof typeof VALID_COLORS
-  >;
+  const theme = useTheme();
+  const colorKeys = Object.keys(theme.COLORS);
 
   return (
     <MainContainer>
-      {colorKeys.map((category) => {
-        const categoryColors = VALID_COLORS[category];
-
+      {colorKeys.map((colorKey) => {
+        const colorHex = theme.COLORS[colorKey] as string;
```

Comment:
This file had particular typing setup to get inference up to the point of needing to cast with `as string`. Since we're only mapping over once now, we no longer need to do that, and should restore the inference we had like so:

<img width="706" alt="Screen Shot 2020-12-03 at 9 31 11 AM" src="https://user-images.githubusercontent.com/13544620/101050826-68fa0580-354a-11eb-9939-bd00963ff805.png">

Reply:
thanks fixed